webapp/controller/ProductsOverviewPage.controller.js

```diff
@@ -1,31 +1,44 @@
-sap.ui.define(["./BaseController", "sap/ui/model/json/JSONModel"], function (BaseController, JSONModel) {
-  "use strict";
+sap.ui.define(
+  [
+    "nadzeya/kireyenka/app/controller/BaseController",
+    "sap/ui/model/json/JSONModel",
+    "nadzeya/kireyenka/app/model/formatter/formatter",
+    "nadzeya/kireyenka/app/model/productModel",
+    "nadzeya/kireyenka/app/model/categoryModel",
+    "nadzeya/kireyenka/app/model/supplierModel",
+  ],
+  function (BaseController, JSONModel, formatter, productModel, categoryModel, supplierModel) {
+    "use strict";
 
-  return BaseController.extend("nadzeya.kireyenka.app.controller.ProductsOverviewPage", {
-    onInit: function () {
-      const oView = this.getView();
-      const oProducts = new JSONModel("model/products.json");
-      const oCategories = new JSONModel("model/categories.json");
-      const oSuppliers = new JSONModel("model/suppliers.json");
-      const oProductsOverviewViewModel = new JSONModel({ selectedProducts: [] });
+    return BaseController.extend("nadzeya.kireyenka.app.controller.ProductsOverviewPage", {
+      deleteProductsButtonTextFormatter: formatter.deleteProductsButtonTextFormatter,
```

Comment:
usually, we just import formatter end use the in view like.<Text text="{..., formatter: 'formatter.getDeleteProductButtonText'}".../>
It's fine when you need one or two functions, but you won't do this for 10 and more functions.